docs/src/modules/components/Demo.js

```diff
@@ -106,6 +106,7 @@ const useStyles = makeStyles(
     /* Isolate the demo with an outline. */
     demoBgOutlined: {
       padding: theme.spacing(3),
+      backgroundColor: theme.palette.background.paper,
```

Comment:
make the demo looks the same as codesandbox when apply darkmode

![image](https://user-images.githubusercontent.com/18292247/120280632-225f4b00-c2e2-11eb-8b43-f060b17dfa2f.png)

Reply:
👍 for the direction taken by the outcome. We render the demo under `theme.palette.background.default`. However, why use the paper design token? Why not apply the same background color to the whole page?

Reply:
![image](https://user-images.githubusercontent.com/18292247/120419124-044e2500-c38c-11eb-9963-09d9a0af081b.png)

Here, if the whole page has the same background, everything blend together and hard to focus. On the other hand, if the demo is the darkest it is just catch the focus instantly.

Reply:
Interesting. I would add that there are different demos variant. We have:

- `"bg": true`: https://deploy-preview-26541--material-ui.netlify.app/components/stack/#usage
- `"bg": "outlined"`, the default, https://deploy-preview-26541--material-ui.netlify.app/components/checkboxes/#basic-checkboxes
- `"bg": "inline"`, https://deploy-preview-26541--material-ui.netlify.app/components/paper/#variants

I almost feel like "bg": true should be the current "bg": "outlined" and "bg": "outlined" should have the bgcolor of "bg": "inline".